logagg/formatters.py

```diff
@@ -48,30 +50,34 @@ def nginx_access(line):
                        }'
     >>> output_line2 = nginx_access(input_line2)
     >>> pprint.pprint(output_line2)
-    {'data': {u'body_bytes_sent': 152.0,
+    {'data': {u'body_bytes_sent': u'152',
               u'http_referer': u'-',
               u'http_user_agent': u'python-requests/2.18.4',
               u'http_x_forwarded_for': u'-',
               u'remote_addr': u'192.158.0.51',
               u'remote_user': u'-',
               u'request': u'POST /mpub?topic=heartbeat HTTP/1.1',
-              u'request_time': 0.0,
-              u'status': 404.0,
+              u'request_time': u'0.000',
+              u'status': u'404',
```

Comment:
type modification must be done only by the formatter function and that too specific to each field without using a general function like str2int or some such.

Reply:
for eg: in nginx:

log['request_time'] = float(log['request_time'])